.circleci/config.yml

```diff
@@ -1,11 +1,20 @@
+# This file is generated automatically by concatenating the files in the .circleci/configuration/ directory.
+# DO NOT EDIT this file directly. Instead, edit the files in the .circleci/configuration/ directory and rely on the pre-commit hook to update this file.
```

Comment:
I was wondering if we should add a test job to make sure this file matches the output from the individual configuration files, to make sure it's not outdated/it's changed accidentally.